### PR TITLE
[ACS-5805] Update mtls.md with separate config for Shared File Store

### PR DESCRIPTION
**Pull request by kmagdziarz**

### CODE REVIEW

**Comment by mstrankowski** [comment]
Just a short explanation of these changes, we’ve ran into an issue, where we’ve previously tried to create a single configuration for 2 things, during tests we’ve found out that the timeout that has been previously set for Shared File Store communication - 5 seconds is heavily restricting the bigger Transform Engine requests, during investigation of a reported issue, we’ve found out that we missed no timeout being set for Transform Engine communication, which essentially meant it is infinite.
To keep our configurations as in-line with previous behaviour as possible, we’ve had to separate the shared file store from other ATS services that have common settings.

File: content-services/latest/config/mtls.md [hunk 1]
```diff
@@ -208,12 +208,21 @@ The default settings for `transform` are shown below:
 httpclient.config.transform.mTLSEnabled=false
 httpclient.config.transform.maxTotalConnections=20
 httpclient.config.transform.maxHostConnections=20
-httpclient.config.transform.socketTimeout=5000
```
Comment (mstrankowski) [inline content-services/latest/config/mtls.md:211]: In that case I'd change "The default settings for `transform` are shown below:" to "The default settings for `transform` are shown below, timeouts are infinite by default:"

Comment (mstrankowski) [reply content-services/latest/config/mtls.md:211]: I've seen more things, I'll provide a fix myself along with a few other changes

**Review by mstrankowski — APPROVED** [review]

**Review by pzhyland — DISMISSED** [review]

**Review by mstrankowski — DISMISSED** [review]

**Review by anxumalo — APPROVED** [review]
@mstrankowski @pzhyland Thanks for the reviews & updates 👍🏼

**Pull request merged**